DuckDuckGo/HomeMessagesStorage.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  HomeMessageStorage.swift
+//  DuckDuckGo
+//
+//  Copyright © 2020 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Core
+
+public class HomeMessageStorage {
```

Comment:
Really appreciate you thinking ahead here and separating out the model.  That'll be useful later.  However, I think should try and reuse `UserDefaultsWrapper`, especially given there's only 1 key at the moment and that we haven't got any tests for this.   

The future model I had in mind would be entirely driven by the JSON on the endpoint so I think just assuming it'll be default browser for now is good enough.